test/formula.jl

```diff
@@ -1,134 +1,125 @@
 @testset "formula" begin
 
-    # TODO:
-    # - grouped variables in formulas with interactions
-    # - is it fast?  Can expand() handle DataFrames?
-    # - deal with intercepts
-    # - implement ^2 for datavector's
-    # - support more transformations with I()?
+    using StatsModels: hasresponse, hasintercept
 
-    ## Formula parsing
-    using StatsModels: @formula, Formula, Terms
+    y, x1, x2, x3, a, b, c, One = term(:y, :x1, :x2, :x3, :a, :b, :c, 1)
```

Comment:
Better use e.g. `onet` than `One` as leading uppercase is generally used for types.